### PR TITLE
[feat] Create fan club

### PR DESCRIPTION
**Pull request by rado196**
### This PR includes the following implementations:
- Create a fan club.
- Some UI updates.

---

##### Some futures _(e.g.  Choose file, Preview file, File upload, etc.)_ have dependencies and need to be implemented in other PR.

### CODE REVIEW

File: clients/app/src/views/screens/CreateFanClub/CreateFanClubNavigation.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import CreateFanClub from './screens/CreateFanClub';
+import SelectCity from './screens/SelectCity';
+import { IScreenType, ICreateFanClubNavigationProps } from './types';
+import SearchListModal from '../SearchListModal';
+
+const Stack = createNativeStackNavigator();
+
+const screens: Array<IScreenType> = [
```
Comment (hay92k) [inline clients/app/src/views/screens/CreateFanClub/CreateFanClubNavigation.tsx:10]: why not in navigation/screens?

File: clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/CreateFanClub.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      fanClubsStore.setAddFieldValue('teamId', route.params.selected.id);
+    }
+  }, [route.params]);
+
+  React.useEffect(() => {
+    if (add.status === 'success') {
+      fanClubsStore.reset();
+      goBack();
+
+      // @TODO: show success message
```
Comment (hay92k) [inline clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/CreateFanClub.tsx:80]: let's push to new fan club screen

**Review by hay92k — CHANGES_REQUESTED** [review]

File: clients/app/src/views/screens/CreateFanClub/CreateFanClubNavigation.tsx (hunk 1 above)
Comment (rado196) [reply clients/app/src/views/screens/CreateFanClub/CreateFanClubNavigation.tsx:10]: will change and push soon.

**Review by hay92k — APPROVED** [review]
lgtm

File: clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/CreateFanClub.tsx (hunk 2 above)
Comment (rado196) [reply clients/app/src/views/screens/CreateFanClub/screens/CreateFanClub/CreateFanClub.tsx:80]: will add with new PR, and we need to do same think with create event.

**Pull request merged**